### PR TITLE
add analytics migration note for 4.0.0 docs

### PR DESCRIPTION
**Pull request by isharac**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-400.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ The following information describes how to upgrade your API Manager server **fro
 !!! Attention
     If you are using WSO2 Identity Server (WSO2 IS) as a Key Manager, follow the instructions in [Upgrading WSO2 IS as the Key Manager to 5.11.0]({{base_path}}/install-and-setup/upgrading-wso2-is-as-key-manager/upgrading-from-is-km-520-to-is-5110) instead of the following steps.
 
+!!! Attention
+    Since on-premise analytics data cannot be migrated to the Cloud, it is required to keep the old analytics server and UI running for required period of time (3 months) after migrating to the new version API-M 4.0.0.
```
[the remark this answers is not in the document]
Comment (isharac) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-400.md:12]: i gave as an example

Comment (Mariangela) [inline en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-400.md:12]:
```suggestion
    As the on-premise analytics data cannot be migrated to the Cloud, you need to maintain the old analytics server and keep the UI running for as long as you need that data (e.g., 3 months) after migrating to the new version of analytics in WSO2 API-M 4.0.0.
```

Comment (Mariangela) [reply en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-200-to-400.md:12]: If the above suggestion is correct. Pls apply it to the other pages as well.

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
PR sync https://github.com/wso2/docs-apim/pull/4044

**Pull request merged**